docs/index.asciidoc

```diff
@@ -175,6 +175,17 @@ Another option is to define patterns _inline_ in the filter using `pattern_defin
 This is mostly for convenience and allows user to define a pattern which can be used just in that 
 filter. This newly defined patterns in `pattern_definitions` will not be available outside of that particular `grok` filter.
 
+[id="plugins-{type}s-{plugin}-ecs"]
+==== Elastic Common Schema (ECS)
+
+To ease migration to ECS the filter plugin is providing 2 sets of pattern files: the old 'legacy' set
+and another set of patterns with captures (event field names) that are compliant with the schema.
+
+The ECS set is a drop-in replacement - has all of the pattern definitions as the legacy files.
+New features and enhancements will be primarily added to ECS compliant files, the legacy patterns might
+still receive bug fixes which are backwards compatible.
+
+To toggle between the two modes consult the <<plugins-{type}s-{plugin}-ecs_compatibility>> setting.
```

Comment:
```suggestion

To ease migration to the {ecs-ref}[Elastic Common Schema (ECS)], the filter
plugin offers a new set of ECS-compliant patterns in addition to the existing
patterns. The new ECS pattern definitions capture event field names that are
compliant with the schema.

The ECS pattern set has all of the pattern definitions from the legacy set, and is
a drop-in replacement. Use the <<plugins-{type}s-{plugin}-ecs_compatibility>>
setting to switch modes. 

New features and enhancements will be added to the ECS-compliant files. The
legacy patterns may still receive bug fixes which are backwards compatible.

```

Reply:
Wording and structure changes for your consideration.

Reply:
Thanks Karen - this reads so much better! :heart_eyes: